server/src/routes/auth.ts

```diff
@@ -144,3 +145,33 @@ auth.get("/verify-token", jwt, async (req, res, next) => {
 		next(e)
 	}
 })
+
+auth.post("/signout", jwt, async (req, res, next) => {
+	try {
+		const authHeader = req.headers["authorization"]
+		const token = authHeader?.split(" ")[1]
+		let reason = ""
+		if (token == null) return res.sendStatus(401)
+		
+		verify(token, config.jwt_secret, (err: any, user: any) => {
+			if (err) return res.sendStatus(403)
+			if(user){
```

Comment:
you should have a space here (`if (user)`) (otherwise it kind of looks like if is a function if())